test/runtests.jl

```diff
@@ -5,5 +5,7 @@ include("test_misc.jl")
 include("test_ops.jl")
 include("test_ad.jl")
 
-# Build the docs
```

Comment:
I think its better to always run them, but only in strict mode on that build bot.